src/ext/property-panel/__tests__/data.test.ts

```diff
@@ -46,6 +49,54 @@ describe('data', () => {
       });
     });
 
+
+    describe('Unsupported Feature on DQ mode:', () => { 
+      let itemData: EngineAPI.IHyperCubeDimensionDef;
+      let args: {
+        properties: {
+          qHyperCubeDef: {
+            qIndentMode: boolean;
+          };
+        };
+      };
+
+      beforeEach(() => {
+        itemData = { qOtherTotalSpec: { qTotalMode: 'TOTAL_EXPR' } } as EngineAPI.IHyperCubeDimensionDef;
+        args = {
+          properties: {
+            qHyperCubeDef: {
+              qIndentMode: false,
+            },
+          },
+        } as Args;
+      });
+
+      afterAll(() => {
+        data = createData({
+          translator: { get: (str) => str },
+          anything: { sense: { isUnsupportedFeature: () => false } },
+        });
+      });
+
+      test('should not show totalMode and totalsLabel when feature is not supported', () => {
+        data = createData({
+          translator: { get: (str) => str },
+          anything: { sense: { isUnsupportedFeature: (f) => f === 'totals' } },
+        });
+        expect(data.items.dimensions.items.totalMode.show).toBe(false);
+        expect(data.items.dimensions.items.totalsLabel.show(itemData, null, args)).toBe(false);
+      });
+
+      test('should show totalsLabel when unsupported feature is not totals', () => {
```

Comment:
This test does not actually add any value. You are just testing the mocked version of `isUnsupportedFeature`. It can safely be removed.